src/spec.c

```diff
@@ -1246,16 +1247,35 @@ void CleanPool_ThreadPoolDestroy() {
   }
 }
 
-size_t CleanPool_WorkingThreadCount() {
-  if (cleanPool) {
-    return redisearch_thpool_num_threads_working(cleanPool);
-  }
+size_t getPendingIndexDrop() {
+  return __atomic_load_n(&pendingIndexDropCount_g, __ATOMIC_RELAXED);
+}
+
+size_t addPendingIndexDrop() {
+  __atomic_fetch_add(&pendingIndexDropCount_g, 1, __ATOMIC_RELAXED);
+}
+
+void removePendingIndexDrop() {
+  uint16_t current = __atomic_load_n(&pendingIndexDropCount_g, __ATOMIC_RELAXED);
+  do {
+    if (current == 0) {
+      return;
+    }
+  } while (!__atomic_compare_exchange_n(&pendingIndexDropCount_g, &current, current - 1, 0, 0, 0));
+
+}
```

Comment:
I wanted to prevent it from wrapping, but as you mentioned, if we return the pending count and display it, it would be visible in FT.INFO if there would be a problem with the count (it would become a very big number) - which is good.